components/Card.js

```diff
@@ -0,0 +1,15 @@
+import { nowPlayingMovies } from "../lib/db";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function MovieCard({ id, image, title, year }) {
```

Comment:
you don't use the `id` and can remove it from the props.

And better to name the Component the same as the file (Card in this case)